setup.py

```diff
@@ -23,7 +23,7 @@
     'boto3>=1.9.6',
     'requests[security]',
     'configparser',
-    'fido2',
+    'fido2>=0.8.1,<0.9.0',
```

Comment:
I've added `<0.9.0` to potentially avoid breaking on the next minor version of fido2.